widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/MenuBarUtils.java

```diff
@@ -75,4 +75,13 @@ public static native DecoratedPopupPanel getPopup( MenuBar menuBar ) /*-{
   public static native MenuBar getParentMenu( MenuBar menuBar ) /*-{
     return menuBar.@com.google.gwt.user.client.ui.MenuBar::parentMenu;
   }-*/;
+
+  /**
+   * Provides access to MenuBar's {@link com.google.gwt.user.client.ui.MenuBar#doItemAction(MenuItem, boolean, boolean)
+   * doItemAction}
+   */
+  public static native void doItemActionLocalSuper( MenuItem item, boolean fireCommand, boolean focus ) /*-{
```

Comment:
In this context, the method could be called simply `doItemAction`, matching the original method name.